WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -653,6 +654,15 @@ class AztecPostViewController: UIViewController, PostEditor {
         navigationItem.rightBarButtonItems = [moreBarButtonItem, publishButton]
     }
 
+    /*
+     This is to restore the navigation bar colors after the UIDocumentPickerViewController has been dismissed,
+     either by uploading media or cancelling. Doing this in the UIDocumentPickerDelegate methods either did nothing
+     or the resetting wasn't permanent.
+     */
+    fileprivate func resetNavigationColors() {
```

Comment:
Nipticky: In Swift we use Xcode's standard comment keyword (`///`), versus the `/*` (ObjC!).

You can add this automatically by positioning the caret over the method name, and pressing **Option** + **CMD** + **/**